AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.ts

```diff
@@ -25,6 +25,15 @@ export class ParticipantListComponent implements OnInit, OnChanges {
 
     constructor(private logger: Logger, private videoHearingsService: VideoHearingsService) {}
 
+    ngDoCheck(): void {
+        const containsNewParticipants = this.hearing.participants?.some(x => !this.sortedParticipants.every(y => y !== x)) ?? false;
+        const containsRemovedParticipants = this.sortedParticipants?.some(x => !this.hearing.participants.every(y => y !== x)) ?? false;
```

Comment:
This is cognitively quite complex. Maybe a quick call to discuss. And look at a way to simplify. Instead of !array.every could you use array.includes?

Reply:
Yeah, makes sense. I have refactored it. Did you still want to have a call?

Reply:
No that's actually fine.